docker/Dockerfile_ubuntu_18.04

```diff
@@ -0,0 +1,27 @@
+FROM nvidia/cudagl:9.2-runtime-ubuntu18.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    python3 python3-dev ipython3 module-init-tools curl build-essential python3-pip
+
+RUN pip3 install -U pip setuptools wheel
+
+RUN pip3 install numpy posix_ipc holodeck pytest opencv-python
+
+RUN adduser --disabled-password --gecos "" holodeckuser
+
+RUN echo holodeckuser:holodeck | chpasswd
```

Comment:
Hmm what happens if we just don't have a password? That would be easier. Since we're improving these images don't be afraid to change things to make it better

Reply:
Removed the password